crates/sui-node/src/main.rs

```diff
@@ -101,8 +119,19 @@ fn main() {
     let node_once_cell_clone = node_once_cell.clone();
     let rpc_runtime = runtimes.json_rpc.handle().clone();
 
+    // let sui-node signal main to shutdown runtimes
+    let (shutdown_tx, shutdown_rx) = mpsc::channel(1);
+
     runtimes.sui_node.spawn(async move {
-        match sui_node::SuiNode::start_async(&config, registry_service, Some(rpc_runtime)).await {
+        match sui_node::SuiNode::start_async(
+            &config,
+            registry_service,
+            Some(rpc_runtime),
+            run_with_range,
+            Some(shutdown_tx),
```

Comment:
shutting down sui-node here makes sense for our validation workflows but not sure if we want this in disaster recovery scenarios?

Reply:
Yeah I think so. My understanding is that in such a case, rather than wanting to somehow signal to the node to make some change, we would be installing a new binary, then restarting the node without the range.